client/state/sites/test/actions.js

```diff
@@ -191,4 +196,56 @@ describe( 'actions', () => {
 			} );
 		} );
 	} );
+
+	describe( 'deleteSite()', () => {
+		useNock( nock => {
+			nock( 'https://public-api.wordpress.com:443' )
+				.persist()
+				.post( '/rest/v1.1/sites/2916284/delete' )
+				.reply( 200, {
+					ID: 2916284
```

Comment:
Yeah, but if we're not using it, it's enough to just return a `200` code with no actual response.